src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -20,20 +21,22 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 	public readonly connectionString: string;
 	public readonly databaseName: string;
 
-	public isConnected: boolean = false;
+	private _parentId: string;
 
-	constructor(databaseName: string, connectionString: string) {
+	constructor(databaseName: string, connectionString: string, parentId: string) {
 		this.databaseName = databaseName;
 		this.connectionString = connectionString;
+		this._parentId = parentId;
 	}
 
 	public get label(): string {
-		if (this.isConnected) {
-			return this.databaseName + " (Connected)";
-		}
 		return this.databaseName;
 	}
 
+	public get description(): string {
```

Comment:
Might be worth considering to add a fullId property to nodes at some point...

Reply:
There is, but it's only at the 'node' level, not the 'treeItem' level

Reply:
Ideally there'd be a way to refresh the entire tree's labels only.

Reply:
Fair enough. Doesn't exist yet